### ISSUE TITLE
Write tutorial for inference using Bioimage models in Galaxy

### ISSUE THREAD

**Issue by kostrykin** [FAIR-imaging/galaxy-image-community#9]
There was a tutorial on that topic already written at BioHackathon Europe 2023: https://docs.google.com/document/d/1kfSw5gVG7H6LvMG-i9rrz8r-910rwFlaN6-Z8WDQmaY/edit

This only needs to be converted to a tutorial for the Galaxy Training Network.

**Comment by beatrizserrano** [comment]
we can close this, right?

**Comment by kostrykin** [comment]
> we can close this, right?

Yes. I'm appending a list of the relevant PRs for future reference:
- https://github.com/galaxyproject/training-material/pull/5919
- https://github.com/galaxyproject/training-material/pull/5955
- https://github.com/galaxyproject/training-material/pull/5957
- https://github.com/galaxyproject/training-material/pull/5958
